src/Core/StackUri.php

```diff
@@ -25,22 +25,15 @@
  */
 class StackUri extends AbstractStack
 {
-    /**
-     * @var string|null
-     */
-    private $baseUrl;
```

Comment:
this was not used anywhere. i guess we can drop it if its not needed.

Reply:
yes, would say so, too